utilities_common/chassis.py

```diff
@@ -16,3 +16,19 @@ def get_chassis_local_interfaces():
                 lst = data[1].split(",")
                 return lst
     return lst
+
+
+def is_smartswitch():
+    return hasattr(device_info, 'is_smartswitch') and device_info.is_smartswitch()
+
+
+def is_dpu():
+    return hasattr(device_info, 'is_dpu') and device_info.is_dpu()
+
+
+def get_num_dpus():
+    return hasattr(device_info, 'get_num_dpus') and device_info.get_num_dpus()
+
+
+def get_dpu_list():
+    return hasattr(device_info, 'get_dpu_list') and device_info.get_dpu_list()
```

Comment:
@vvolam are you trying to return boolean value?

Reply:
Fixed it. Thank you!